tests/integration.coffee

```diff
@@ -946,8 +946,12 @@ describe 'SDK Integration Tests', ->
 
 					describe 'resin.models.device.getDeviceUrl()', ->
 
+						it 'should eventually be a url containing full length uuid', ->
```

Comment:
Maybe the title can be more like "should eventually return the correct device url given a shorter uuid", or something like that that highlights the important point of the test is the shorter uuid behaviour?